miden-lib/asm/miden/tx.masm

```diff
@@ -53,39 +53,39 @@ export.get_output_notes_hash
     # => [COM]
 end
 
-#! Creates a new note and returns a pointer to the memory address at which the note is stored.
+#! Creates a new note and returns the index of the note.
 #!
 #! Inputs: [tag, note_type, RECIPIENT]
-#! Outputs: [note_ptr]
+#! Outputs: [note_idx]
 #!
 #! tag is the tag to be included in the note.
 #! note_type is the storage type of the note
 #! RECIPIENT is the recipient of the note.
-#! note_ptr is the pointer to the memory address at which the note is stored.
+#! note_idx is the index of the the note to which the asset is added.
 export.create_note
     syscall.create_note
-    # => [note_ptr, ZERO, 0]
+    # => [note_idx, ZERO, 0]
 
     # clear the padding from the kernel response
     movdn.4 dropw swap drop
-    # => [note_ptr]
+    # => [note_idx]
 end
 
-#! Adds an ASSET to the specified note and returns a pointer to the memory address
-#! at which the note is stored.
+#! Adds an ASSET to the specified note and returns the index index of the note
+#! at which the asset is added.
```

Comment:
typo: "index index"

Also: it probably should be "index of the note **to** which the asset is added".